src/Raven.Server/Documents/Archive/DocumentsArchiver.cs

```diff
@@ -0,0 +1,208 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Raven.Client.Documents.Operations.Archival;
+using Raven.Client.ServerWide;
+using Raven.Server.Background;
+using Raven.Server.Documents.TransactionMerger.Commands;
+using Raven.Server.NotificationCenter.Notifications;
+using Raven.Server.ServerWide.Context;
+using Sparrow.Json;
+using Sparrow.Logging;
+using Sparrow.Platform;
+using Voron;
+
+namespace Raven.Server.Documents.Archival
+{
+    public class DocumentsArchiver : BackgroundWorkBase
```

Comment:
Maybe `DataArchiver` since it is gonna deal with with counters, time series etc as well?

Reply:
https://github.com/ravendb/ravendb/pull/15715/commits/17c5028e4ff287e2e13046141cd3c802dd159836

Reply:
Changed to DataArchivist